Ajout guide d'accompagnement de nouveau contributeurs⋅trices (#1216)
YunoHost/doc: doc_writing_guide.md

> # Guide to writing documentation

Reviewer: ```suggestion
# Guide to writing application documentation

## Pages for user / administative documentation

Add a one-click installation button (like for example: https://yunohost.org/#/app_piwigo)

Classification of the application can be done using the available tags (genre, Git, management association, e-mails, etc.)

*/Find a suitable documentation license. Permissive or not? (CC BY_SA)/*

## Clarify the types of uses it has (special and general application (editorial choice)

 + For links to other languages, add (en)` for Egnlish, and so on.

### Things to include in application documentation

 1. An index of what is included, without reiterating the full content of what is to be found in subsequent chapters.
 1. A one-click install button.
 1. A general presentation of the application and its function.
 1. A section detailing administration of the application.
 1. A further section on changes specific to YunoHost.
 1. A section about the desktop client (if applicable). Links to different third party applications (if they exist), (or a link to a list of applications [framalibre.org](https://framalibre.org)) or a link to a page that has to do with the different official applications (if applicable).
 + A further section:
    - Links to the official website.
    - Links to the package in YunoHost.
 1. Documentation of any further uses (documentation of an application).

## Roadmap

1. Documentation of the applications.
   1. Document how the application operates in use (marked: work) level 8/7/6.
   1. Translate the documentation at-least into French and English.
   1. Send a PR to the repository of the app that details its documentation.

## Document an application

Do you need to detail how it works?

```
Is there documentation of the application already present in French and English?
                                                      /     \
                                                     /       \
                                                    /         \
                                                   /           \
                                                  /             \
                                             -------            -------
                                             | Yes |            | No  |----> Document the application  <----<----<----<-----
                                             -------            -------   in the YunoHost documentation.                   |
                                               |                                                                           |
                                          Is the documentation up to scratch in terms of quality?                          |
                                              /  \                                                                         |
                                             /    \                                                                        |
                                            /      \                                                                       |
                                           /        \                                                                      |
                                          /          \                                                                     |
                                      -------      -------                                                                 |
   Direct users to its       ------- | Yes |      | No  |---->---->---->---->---->---->---->---->---->---->---->---->---->-
documentation from the YunoHost      -------      -------
equivalent.
```

## Potential problems you could face.

Is the app compartmentalized (like Nextcloud, which presents a myriad of uses)?
```

# Guide to writing application documentation

## User documentation pages / administrateurs⋅trices

Add a one-click install button (such as https://yunohost.org/#/app_piwigo_fr) and a button on the application integration level.

> Classification of available applications by tags (genre, Git, association management, emails, etc).

Reviewer: ```suggestion
Classification of available applications by tags (genre, Git, association management, e-mails, etc).
```

## Some typical and general uses (writing framework)

 + When a link points to a page that is not in the language of the original page, it is customary to add `(en)` (for a link that points to a page in English).
 + rename the images in the following order:`description_application.ext`.

### General frame application documentation

 1. Logo (dimension 80 pixels high) + level 1 title.
 1. One-click install button, Integration level for each type of processor.
 1. An index at the top of the documentation with cross-references to all the chapters of the documentation.
 1. A general presentation of the application and its function.
 2. A configuration part of the application.
 1. An administration part of the application.
 1. A part on limitations related to YunoHost.
 1. A part on desktop clients (if any). A link to different third-party applications if there are several (possible link to the applications catalgue [framalibre.org](https://framalibre.org)) or a link to the page about desktop applications if official applications are provided.

>  1. A part with :

Reviewer: ```suggestion
 1. A part with:
```
